Makefile

```diff
@@ -20,41 +21,48 @@ deps:           ## [composer yarn assets-dev perm] Install the project PHP and J
 deps: composer yarn assets-dev perm
 
 composer:       ## Install the project PHP dependencies
-	$(APP) composer install
+	$(TOOLS) composer install
 
 yarn:           ## Install the project JS dependencies
 	$(TOOLS) yarn install
 
 db-create:      ## Create the database and load the fixtures in it
-	$(APP) php -r "for(;;){if(@fsockopen('db',3306)){break;}}" # Wait for MariaDB
+	$(TOOLS) php -r "for(;;){if(@fsockopen('db',3306)){break;}}" # Wait for MariaDB
 	$(CONSOLE) doctrine:database:drop --force --if-exists
 	$(CONSOLE) doctrine:database:create --if-not-exists
-	$(CONSOLE) doctrine:schema:create
+	$(CONSOLE) doctrine:migrations:migrate -n
 
 db-fixtures:    ## Reloads the data fixtures for the dev environment
 	$(CONSOLE) doctrine:fixtures:load -n
 
 db-update:      ## Update the database structure according to the last changes
-	$(CONSOLE) doctrine:schema:update --force
+	$(CONSOLE) doctrine:migration:migrate -n
+
+db-diff:
```

Comment:
The comment of other commands (`## Update the database structure according to the last changes`) is used to generate the help. Can you add one here :) ?